contributing/code/bugs.rst

```diff
@@ -32,9 +33,16 @@ If your problem definitely looks like a bug, report it using the official bug
 * Give as much detail as possible about your environment (OS, PHP version,
   Symfony version, enabled extensions, ...);
 
+* If you want to provide a stack trace, be sure to provide the plain text
+  version, which should appear at the bottom of the page. *Do not* provide it
+  as a screenshot, since search engines will not be able to index the text
+  inside them. If the stack trace is long, consider enclosing it in a
+  ``<details>`` html tag.
```

Comment:
Correct, I meant that in the general sense, pasting stack trace somewhere need to be done carefully